lightkurve/seismology/core.py

```diff
@@ -276,11 +353,170 @@ def plot_echelle(self, deltanu=None, numax=None,
 
             cbar.set_label(ylabel)
             ax.set_xlabel(r'Frequency mod. {:.2f}'.format(deltanu))
```

Comment:
We should make sure the unit is done properly in LaTeX + is consistent with our other plots, e.g.

```
axis_label = rf" Frequency mod. {dnu.value:.3f} [{deltanu.unit.to_string('latex')}]"
```

Reply:
Bokeh can't render latex, so I've omitted this.